### PR TITLE
Epmlabsbrn-1064 API for check if user has statistics

### PR DESCRIPTION
**Pull request by andrsam**
[EPMLABSBRN-1064](https://jira.epam.com/jira/browse/EPMLABSBRN-1064)

**API for check if user has statistics**:

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/controller/StudyHistoryControllerV2.kt [hunk 1]
```diff
@@ -26,4 +28,10 @@ class StudyHistoryControllerV2(
         @RequestParam("to", required = true) to: LocalDateTime
     ) = ResponseEntity.ok()
         .body(BaseResponse(data = studyHistoryService.getHistoriesForCurrentUser(from, to)))
+
+    @GetMapping("/user/{userId}/has/statistics")
```
Comment (Dmitriy1Fokin) [inline src/main/kotlin/com/epam/brn/controller/StudyHistoryControllerV2.kt:32]: Please add a new endpoint to the api-contract

**Comment by Shterneregen** [comment]
I see that your pipeline is also failed with Metaspace cause (as mine), I solved it by adding `org.gradle.jvmargs=-XX:MaxMetaspaceSize=512m` in `gradle.properties` file. You can try the same if you want.
image

File: src/main/kotlin/com/epam/brn/repo/StudyHistoryRepository.kt [hunk 2]
```diff
@@ -126,4 +126,10 @@ interface StudyHistoryRepository : CrudRepository<StudyHistory, Long> {
             "AND s.userAccount.id = :userId"
     )
     fun getDayStatistic(userId: Long, year: Int, month: Int, day: Int): List<StudyHistory>
+
+    @Query(
+        "select case when count (s) > 0 then true else false end " +
+            "from StudyHistory s where s.userAccount.id = :userId"
+    )
+    fun isUserHasStatistics(userId: Long): Boolean
```
Comment (Dmitriy1Fokin) [inline src/main/kotlin/com/epam/brn/repo/StudyHistoryRepository.kt:134]: you can make it easier:
1. `@Query("select count (s) > 0 from StudyHistory s where s.userAccount.id = :userId")`
or
2. ` fun existsByBlaBla(userId: Long): Boolean`

File: src/main/kotlin/com/epam/brn/controller/StudyHistoryControllerV2.kt (hunk 1 above)
Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/controller/StudyHistoryControllerV2.kt:32]: why not simplier 
/has/statistics -> /hasStatistics ?

**Pull request merged**